.github/monorepo/split-repositories.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+SPLIT_BRANCH=$1
+REMOTE_TEMPLATE=$2
+FORCE=${3:-false}
+
+set -u
+
+# Import functions
+. $(dirname "$0")/monorepo_functions.sh
+
+assert_split_branch_variable
+assert_remote_template_variable
+
+if [[ "$FORCE" == true ]]; then
+    assert_split_branch_is_not_protected
+fi
+
+echo -e "${BLUE}Splitting branch '$SPLIT_BRANCH'...${NC}"
+
+WORKSPACE=`pwd`
+
+if [[ "$FORCE" == true ]]; then
+    PUSH_OPTS="--force"
+else
+    PUSH_OPTS="--tags"
```

Comment:
Are you sure you want to push all tags always? And that you do not want to push during force?

Reply:
Quite. This option separates two workflows – feature branches (mg-fix-bug, ...) and any of main branches (master, 13.0, ...). 

If we split any of the main branches, we push the tags. When no new tag is introduced, nothing extra happens. When a new tag is created in the process (during the release process), this new tag is propagated. 

If we split the feature branch, we don't expect new tags, but we do not care about the history of branches in packages  – thus `--force`